functions/openhab.bash

```diff
@@ -299,3 +299,17 @@ dashboard_add_tile() {
 
   if echo -e "\\norg.openhab.core.ui.tiles:${application}-link-name=${tileDesc}\\norg.openhab.core.ui.tiles:${application}-link-url=${tileURL}\\norg.openhab.core.ui.tiles:${application}-link-imageurl=${tileImg}" >> "$dashboardConfig"; then echo "OK"; else echo "FAILED"; return 1; fi
 }
+
+## Replace outdated bintray repo
+##
+##    fix_openhab_repo
+##
+fix_openhab_repo() {
+  local warningText="Due to bintray shutdown of services, there is an immediate need to change the openhab stable repository configuration.\\nBintray repo in /etc/apt/sourceslist.d/openhab.list needs to be replaced by Artifactory.\\n\\nDo you want this to be replaced now ?"
```

Comment:
Proper English says that the '?' goes without a space after the end of a sentence.

Reply:
well, then